docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
     environment:
       MYSQL_ROOT_PASSWORD: prestashop
       MYSQL_DATABASE: prestashop
-    restart: always
+    restart: 'no'
```

Comment:
Why ?

Reply:
It's to prevent container from starting with docker. prestashop-git container doesn't, mysql also shouldn't. AFAIK `restart: 'no'` is the default so might as well just remove this line.

Reply:
Actually I like the idea of having the mysql running no matter what if the web container is not running. I'm currently using prestashop with docker, but without the web image.

Perhaps this needs to be discussed in a separate PR, so we don't block this one ?

Reply:
OK. It's not required since mysql uses random port on the host anyway.